contrib/python-werkzeug/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "python-werkzeug"
+pkgver = "3.0.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-flit_core",
+]
+checkdepends = [
+    "python-markupsafe",
+    "python-pytest",
+    "python-pytest-timeout",
+    "python-requests",
+]
+depends = [
+    "python-markupsafe",
+]
```

Comment:
```suggestion
depends = ["python-markupsafe"]
```

single line for 1 dep